testing/trino-test-jdbc-compatibility-old-driver/pom.xml

```diff
@@ -14,7 +14,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.presto-jdbc-under-test>${project.version}</dep.presto-jdbc-under-test>
+        <dep.presto-jdbc-under-test>420-SNAPSHOT</dep.presto-jdbc-under-test>
```

Comment:
How are we going to update this?

Reply:
It's updated automatically by the release plugin. I'll add a comment so that we don't make this same mistake in the future.